src/index.tsx

```diff
@@ -21,11 +21,10 @@ import { renderConsoleArt } from './lib/consoleArt'
 import { reportWebVitals } from './lib/reportWebVitals'
 import { httpClientIntegration } from './utils/sentry/httpclient'
 
-const mode = process.env.MODE ?? process.env.NODE_ENV
-const enableReactScan = mode === 'development' && window.location.hostname === 'localhost'
```

Comment:
Also removed mode as a check here, localhost is all that matters, we may want to run this with builds locally too